### PR TITLE
fastembed integration: add example

### PR DESCRIPTION
**Pull request by nickprock**
I added a simple example about the integration.

related issue #358

### CODE REVIEW

**Comment by masci** [comment]
@nickprock can you run `hatch run lint:fmt` from the integration folder? That should solve the formatting issues

File: integrations/fastembed/example/example.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+query_pipeline = Pipeline()
+query_pipeline.add_component("text_embedder", FastembedTextEmbedder())
+query_pipeline.add_component("retriever", InMemoryEmbeddingRetriever(document_store=document_store))
+query_pipeline.connect("text_embedder.embedding", "retriever.query_embedding")
+
+query = "Who supports fastembed?"
+
+result = query_pipeline.run({"text_embedder": {"text": query}})
+
+print(result['retriever']['documents'][0])
```
Comment (masci) [inline integrations/fastembed/example/example.py:34]:
```suggestion
print(result['retriever']['documents'][0])  # noqa: T201
```

**Review by masci — CHANGES_REQUESTED** [review]

File: integrations/fastembed/example/example.py (hunk 1 above)
Comment (nickprock) [reply integrations/fastembed/example/example.py]: thanks @masci , done

**Review by masci — APPROVED** [review]
All good, sorry for the latency!

**Pull request merged**